layouts/page/ecosystem.html

```diff
@@ -160,6 +160,18 @@ <h3 class="text-white">libcuspatial</h3>
           <a href="https://docs.rapids.ai/api/libcuspatial/stable" target="_blank">Documentation <i
               class="fa-solid fa-arrow-up-right"></i> </a>
         </div>
+        <div class="col-md-4 py-3">
+          <h2 class="text-white"><i class="fa-regular fa-terminal"></i> cuVS </h2>
+          <div class="tags-container border-top border-2 border-white">
+            <span class="tags-item text-white border-white"><i class="fa-solid fa-tag"></i> VECTOR SEARCH </span>
+            <span class="tags-item text-white border-white"><i class="fa-solid fa-tag"></i> PYTHON / C / C++ / RUST </span>
+          </div>
+          <p class="text-white"> The cuVS library contains optimized algorithms for approximate nearest neighbors and clustering, along with many other essential tools for accelerated vector search. Integrate cuVS into any vector database or vectorized data application for optimized speed and efficiency in information retrieval.</p>
+          <a href="https://github.com/rapidsai/cuvs" target="_blank">GitHub <i
+              class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/raft/stable/" target="_blank">Documentation <i
```

Comment:
Should this go to https://docs.rapids.ai/api/cuvs/nightly/?

Reply:
Still in transition - there is a note in the docs about the migration.